src/Icon/README.md

```diff
@@ -26,6 +26,10 @@ HTML attributes can be passed to this component allowing for customization of th
 // import { Add, AddCircle } from '@edx/paragon/icons';
 <div className="d-flex align-items-center bg-dark">
   <Icon src={Add} className="mx-3 text-white" />
+  <Icon src={Add} className="mx-3 text-white pgn__icon__xs" />
+  <Icon src={Add} className="mx-3 text-white pgn__icon__sm" />
+  <Icon src={Add} className="mx-3 text-white pgn__icon__md" />
+  <Icon src={Add} className="mx-3 text-white pgn__icon__lg" />
```

Comment:
@keithgg While I still this will be a valuable and welcome addition, I think you could also alternatively try using Paragon's `Hyperlink` component prop for [your use case](https://github.com/openedx/frontend-lib-content-components/pull/275):

```jsx
<Hyperlink className="x-small" destination="https://www.edx.org" target="_blank">
  YouTube video
</Hyperlink>
```

Reply:
[inform] I also brought this up at today's Paragon Working Group meeting (see [notes](https://openedx.atlassian.net/wiki/spaces/BPL/pages/3707797533/2023-03-22+Meeting+notes)). No concerns from the design side of things with introducing a new `xs` size variant of 1rem for `Icon` component 👍

Reply:
Awesome stuff @adamstankiewicz! I've updated the PR with the new prop (and switch my original PR to use the `Hyperlink` component). Please review when you get a chance. 

If you're happy with the implementation, I'll add some tests so that the coverage doesn't decrease.